app/ui/src/app/customizations/api-connector/api-connector-list.component.html

```diff
@@ -2,20 +2,20 @@
   <div class="row">
     <div class="col-xs-12">
       <h2>API Client Connector</h2>
-      <p>Nullam quis risus eget urna mollis ornare vel eu leo. Find out more at <a>{{ config.getSettings('branding', 'appName', 'Syndesis') }} Help</a></p>
+      <p>Nullam quis risus eget urna mollis ornare vel eu leo. Find out more at <a>{{ config.getSettings('branding', 'appName', 'Syndesis') }} Help</a>.</p>
```

Comment:
Please do not bind a function call into an Angular interpolation. Otherwise you'll wind up calling `config.getSettings('branding', 'appName', 'Syndesis')` on each cycle of the change detector loop.

Tip: add this to a class field initialized at the `OnInit` stage.

Reply:
We discussed this before. Totally agree, and I had originally fixed this. Let me update it now. 👍